README.md

```diff
@@ -1,17 +1,20 @@
 # Lisk SDK examples
 
-Example applications using Alpha SDK.
+This project contains example blockchain applications using the latest Lisk SDK.
+Each example project represents an industry use case in which we showcase the potential of the Lisk SDK and how it can transform this industry.
 
-## [Transport Workshop](./transport/Workshop2.adoc)
-Follow the Lisk Transport workshop which has been designed for the **Lisk.js 2019** event!
+Currently, we have two projects ready:
 
-## [Hello World](./hello_world/README.md)
-A hello world application of Alpha SDK. Guides through the basic flow of using Alpha SDK. By creating a new custom transaction with no practical use case, you can get an idea of why and how to implement the required methods of the custom transaction interface.
+- Hello world (beginner example)
+- Lisk Transport (**Supply chain industry use case**)
 
-## [Cashback](./cashback/README.md)
+All industry use cases will be kept up to date with the latest Lisk SDK release.
+Also, you'll find an `/archive` folder that contains older examples referring to previous Lisk SDK releases. 
+At the moment, the `/archive` folder contains examples for Lisk SDK `version 2.3.5`.
```

Comment:
```suggestion
At the moment, the `/archive` folder contains examples for Lisk SDK `version 2.x`.
```
I would leave this more general, for example our latest 2.x version of the Lisk SDK is `2.3.7` and most tutorials should work with that version as well